manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/persistence/models/Bridge.java

```diff
@@ -24,6 +24,8 @@
                 query = "from Bridge_V2 where name=:name and customer_id=:customerId"),
         @NamedQuery(name = "BRIDGE_V2.findByIdWithConditions",
                 query = "from Bridge_V2 b left join fetch b.conditions where b.id=:id"),
+        @NamedQuery(name = "BRIDGE.findByIdAndCustomerIdWithConditions",
+                query = "from Bridge_V2 b left join fetch b.conditions where b.id=:id and customer_id=:customerId"),
```

Comment:
We might move here the `distinct` (and remove the `firstResult`) or make a comment to remember that this query returns duplicates.